### ISSUE TITLE
Problems with main.py

### ISSUE THREAD

**Issue by zedabotinha** [J3ldo/UGC-Sniper#28]
I been trying for HOURS to make this work, every time i try to open main.py it just crashes immediatly. I tried EVERYTHING, installing the requests thing, reinstalling it, hell i even tried putting proxies on it to see if it works
I even tried to execute it with the command prompt but it also doesn't work, it keeps saying "KeyError: 'id'" whatever that means
Also, i put my correct cookie in, i copy and pasted it 6 times.

**Comment by murderee** [comment]
Hey, are you sure there are no extra spaces after your cookie?

**Comment by murderee** [comment]
and in main.py, in IDLE replace cookie.txt and limiteds.txt with their full path names

**Comment by EppoV** [comment]
Idk how to fix this but every time when i open the main thing. When i type or click it jsut crashes.
pls someone help me.

**Comment by zedabotinha** [comment]
> and in main.py, in IDLE replace cookie.txt and limiteds.txt with their full path names

Yes i also tried that, but it just gives me a error every time i try it

![Capture](https://user-images.githubusercontent.com/44984369/231285257-c8376460-babe-414e-a04d-874ea3bc7639.PNG)
This is how i put the paths in

**Comment by fallino11** [comment]
You need to install the requests library.
Here is an advanced guide: https://www.geeksforgeeks.org/how-to-install-requests-in-python-for-windows-linux-mac/
If you have multiple python version, please specify which python version you are using. (According to your Screenshot Python 3.11.3)
Try with: `python -3.11.3 -m pip install requests`
If this doesn't work for you, please give us more information to work with.
There should be no issue with the code.

**Comment by murderee** [comment]
> You need to install the requests library. Here is an advanced guide: https://www.geeksforgeeks.org/how-to-install-requests-in-python-for-windows-linux-mac/ If you have multiple python version, please specify which python version you are using. (According to your Screenshot Python 3.11.3) Try with: `python -3.11.3 -m pip install requests` If this doesn't work for you, please give us more information to work with. There should be no issue with the code.

really late reply, but if that does not work just try `pip3 install requests`
